Backpack/FlareView/Classes/BPKFlarePath.h

```diff
@@ -52,11 +42,12 @@ IB_DESIGNABLE @interface BPKFlarePath : NSObject
  * Creates and returns a UIBeizerPath of the flare mask for a given size.
  *
  * @param size CGSize of the path required.
+ * @param flareSize CGSize of the path required.
+ * @param flarePosition BPKFlarePosition of the path required.
  */
 + (UIBezierPath *)flareViewPathForSize:(CGSize)size
-                         flarePosition:(BPKFlarePosition)flarePosition
-    NS_SWIFT_NAME(makeFlarePath(size:flarePosition:));
```

Comment:
removed the NSSwiftName because this class is not exposed to consumers, so it'll never be called from swift anyway

Reply:
Doesn't it make sense to have it anyway? It does no harm and if we eventually expose it we won't forget to add a swift name